tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -68,12 +68,15 @@ def start_node(self, node, timeout_sec=60, **kwargs):
 
         node.account.mkdirs(OpenMessagingBenchmarkWorkers.PERSISTENT_ROOT)
 
-        start_cmd = f"cd /opt/openmessaging-benchmark; \
-                      HEAP_OPTS=\" \" \
-                      bin/benchmark-worker \
-                      --port {OpenMessagingBenchmarkWorkers.PORT} \
-                      --stats-port {OpenMessagingBenchmarkWorkers.STATS_PORT} \
-                      >> {OpenMessagingBenchmarkWorkers.STDOUT_STDERR_CAPTURE} 2>&1 & disown"
+        start_cmd = (
+            f"cd /opt/openmessaging-benchmark; "
+            f"HEAP_OPTS=\" \" "
+            f"KAFKA_OPTS=\" \" "
```

Comment:
This 1 line is actual material change. The redpanda `main` OMB branch has KAFKA_OPTS use now, so once we update the SHA this will break, fixing this preemptively.